File: Python/Monitor.py
from Elektronik import Elektronik

class Monitor(Elektronik):
    def __init__(self, id=0, nama="", merek="", harga="", gambar="",
                 ukuran_layar="", layar="", resolusi=""):
        super().__init__(id, nama, merek, harga, gambar)
        self.ukuran_layar = ukuran_layar
        self.layar = layar
        self.resolusi = resolusi

File: Java/Monitor.java
public class Monitor extends Elektronik {
    protected String ukuranLayar;
    protected String layar;
    protected String resolusi;

    public Monitor() {
        super();
        this.ukuranLayar = "";
        this.layar = "";
        this.resolusi = "";
    }

    public Monitor(int id, String nama, String merek, String harga, String gambar,
                   String ukuranLayar, String layar, String resolusi) {
        super(id, nama, merek, harga, gambar);
        this.ukuranLayar = ukuranLayar;
        this.layar = layar;
        this.resolusi = resolusi;
    }
}
